packages/widget-notes/src/Context.tsx

```diff
@@ -76,6 +78,12 @@ const NoteProvider = ({ children }: { children: React.ReactElement }) => {
     globalNotes[index] = note
     setNotes(globalNotes)
   }
+  const _isInterestedInSyncFeature = (interested: boolean) => {
+    if (interested) {
+      return eventListener.emit('telemetryEvent', { eventName: 'yesNoteSyncInterest' })
+    }
+    eventListener.emit('telemetryEvent', { eventName: 'noNoteSyncInterest' })
```

Comment:
This would make it easier to filter for in Azure

```suggestion
      return eventListener.emit('telemetryEvent', { eventName: 'syncInterestNoteYes' })
    }
    eventListener.emit('telemetryEvent', { eventName: 'syncInterestNoteNo' })
```